### ISSUE TITLE
Not await removes brackets to become invalid code

### ISSUE THREAD

**Issue by drekka** [nicklockwood/SwiftFormat#1327]
If I have an await'd Boolean expression which I'm "notting" like this:

```swift
if !(await isSomething()) { ...
```

Swift format removes the brackets to produce this (which won't compile):

```swift
if !await isSomething() { ...
```

I can get around it with:

```swift
if await isSomething() == false { ...
```

But I'd rather tweak a SwiftFormat setting to somehow ignore the brackets, or have a rule added that says something like if the parser sees `<!-operator><opening-bracket><await><space><text><closing-bracket>` then it should leave the brackets in place because they are giving scope to the "!" not operator. 

Or something like that :-)

**Comment by nicklockwood** [comment]
Thanks for reporting. This is definitely a bug, so I'll get that fixed. In the meantime, does rewriting it like this work?

```swift
if await !isSomething() { ... }
```

**Comment by drekka** [comment]
Oh dammit. Now I feel like a real id10t! 

Yes that works just fine. Thanks.

**Comment by facumenzella** [comment]
[This](https://github.com/nicklockwood/SwiftFormat/pull/1334) should fix it

**Comment by nicklockwood** [comment]
Thanks! Fix landed in 0.50.7
